lib/config.js

```diff
@@ -37,20 +37,29 @@ const DEFAULT_OPTIONS = {
 const _config = {};
 
 
+function mergeOptions(optionsA, optionsB) {
```

Comment:
Up to you but maybe this can be simplified a bit for clarity if argv can work with empty input (which I would assume it does). I also made a slight change since usually you are applying a change from one dictionary onto the other in the names of vars:
```
function mergeOptions(defaultOptions, optionsToMerge) {
    return Object.assign(argv(CLI_OPTIONS, defaultOptions),
                                     argv(CLI_OPTIONS, optionsToMerge));
}
```